analysis/generate_report.py

```diff
@@ -157,6 +157,8 @@ def generate_report(experiment_names,
     else:
         experiment_df = queries.get_experiment_data(experiment_names)
 
+    description = queries.get_experiment_description(experiment_names[0])
```

Comment:
nit: move experiment_names[0] into a local main_experiment and use both at line 150 and 160.

Reply:
Done.